modules/end-user-guide/partials/proc_building-an-image-for-intellij-idea-community-or-pycharm-community.adoc

```diff
@@ -0,0 +1,99 @@
+= Building an image for IntelliJ IDEA Community or PyCharm Community
+
+This procedure describes how to build an image for IntelliJ IDEA Community or PyCharm Community.
+
+.Prerequisites
+
+* The build host has at least 2 GB of available RAM.
+
+* The build host has an installation of following tools:
+
+** link:https://www.docker.com/[Docker] version 18.09 or greater, supporting link:https://docs.docker.com/develop/develop-images/build_enhancements/[Docker BuildKit]
+
+** link:https://git-scm.com[Git]
+
+** link:https://www.gnu.org/software/libc/manual/html_node/Getopt.html[GNU getopt]
+
+** link:https://www.gnu.org/software/wget/[GNU wget]
+
+**  link:https://openjdk.java.net/projects/jdk/11/[Java Development Kit (JDK) version 11]
+
+** link:https://stedolan.github.io/jq/[jq]
+
+
+.Procedure
+
+. Get a local copy of the link:https://github.com/che-incubator/jetbrains-editor-images[JetBrains Projector Editor Images repository].
++
+----
+$ git clone https://github.com/che-incubator/jetbrains-editor-images
+$ cd jetbrains-editor-images
+----
+
+. Run the build script and select the IDE package and package version:
++
+----
+$ ./projector.sh build
+[info] Select the IDE package to build (default is 'IntelliJ IDEA Community'):
+       1) IntelliJ IDEA Community
+       2) PyCharm Community
+[info] Select the IDE package version to build (default is '2020.3.3'):
+       1) 2020.3.3
+       2) 2020.3.2
+       3) 2020.3.1
+----
+
+. To test the image, run it locally and navigate to `++http://localhost:8887++` to access the IDE.
++
+----
+$ ./projector.sh run
+----
+
+. Publish the image to a registry accessible by {prod-short}, and remember the location: __<image>__.
+
+. Create a `+meta.yaml+` file with the following content:
++
+[source,yaml,subs="+quotes,macros,attributes"]
+----
+apiVersion: v2
+publisher: __<publisher>__ <1>
+name: intellij-ide
+version: latest
+type: Che Editor
+displayName: IntelliJ Platform IDE
+title: IDE based on the IntelliJ Platform as Editor for Eclipse Che
+description: IDE based on the IntelliJ Platform running using Projector
+icon: https://www.jetbrains.com/apple-touch-icon.png
+category: Editor
+repository: https://github.com/che-incubator/jetbrains-editor-images
+firstPublicationDate: "2021-04-10"
+spec:
+  endpoints:
+   -  name: "intellij"
+      public: true
+      targetPort: 8887
+      attributes:
+        protocol: http
+        type: ide
+        path: /projector-client/index.html?backgroundColor=434343&wss
+  containers:
+   - name: intellij-ide
+     image: "__<image>__" <2>
+     mountSources: true
+     volumes:
+         - mountPath: "/home/projector-user"
+           name: projector-user
+     ports:
+         - exposedPort: 8887
+     memoryLimit: "4096M"
+----
+<1> __<publisher>__: Your publisher name.
+<2> __<image>__: Location of the IDE image in a registry accessible by {prod-short}.
+
+. Publish the `meta.yaml` file to an HTTPS resource accessible by {prod-short}: `__<URL_to_meta.yaml>__`.
```

Comment:
Looks like `__<URL_to_meta.yaml>__` is an orphan here.

Reply:
```suggestion
. Publish the `meta.yaml` file to an HTTPS resource accessible by {prod-short} and copy the full path to the file for use as `__<URL_to_meta.yaml>__` when configuring a workspace devfile for this IDE image.
```

Reply:
It's not necessary to 'copy'.